frontstage/templates/register/register-pending-survey-enter-your-details.html

```diff
@@ -56,7 +56,11 @@ <h1 class="u-fs-xl">Create an account</h1>
         {{ form.batch_no }}
         {{ form.is_transfer }}
         <h2>Your name</h2>
-
+        {% if errors %}
```

Comment:
Does this need to only happen if there is an error?  What happens if you just get the data from an empty form?

Reply:
We need to retain the data on the UI after there is an error. So we will land up on the same view only in case of error.